pip/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-name = "qsharp_python"
-version = "0.0.0"
+name = "qsharp"
```

Comment:
I thought we were going with `qsharp-preview` to avoid any risk with the existing `qsharp` package for now. CC @minestarks

Reply:
I set up a script which can apply the nightly or preview suffix to the package names. Unless you want to default to qsharp-preview for all packages, and then change the whole name to qsharp-nigthly and qsharp.

Reply:
I see in the npm package.json and jupyterlabs package.json you have the `-nightly` suffix on the name. Why the difference here?

Reply:
This name is ignored when the package is built for publishing (through `pip` and not Cargo). It's only ever "seen" when you run `cargo build`. It could be anything that doesn't conflict with an existing Cargo package name in this repo. 

We have questions about this name literally every time this file is touched in a PR, and it's already been changed back-and-forth between `qsharp` and `qsharp_python` a couple times. I don't think we'll be able to find the magic name to eliminate all confusion :D